src/pages/BrowseData.js

```diff
@@ -0,0 +1,277 @@
+import { Fragment } from 'react'
+import { div, h, img, p, span } from 'react-hyperscript-helpers'
+import { pure } from 'recompose'
+import { buttonPrimary, Clickable, link } from 'src/components/common'
+import { InfoBox } from 'src/components/PopupTrigger'
+import TooltipTrigger from 'src/components/TooltipTrigger'
+import { TopBar } from 'src/components/TopBar'
+import amppdLogo from 'src/images/browse-data/Amp@2x.png'
+import cancerCell from 'src/images/browse-data/browse-data-cancer-cell.jpg'
+import gtexLogo from 'src/images/browse-data/GTeX@2x.png'
+import hcaLogo from 'src/images/browse-data/HCA@2x.png'
+import nhsLogo from 'src/images/browse-data/NHS@2x.png'
+import topMedLogo from 'src/images/browse-data/TopMed@2x.png'
+import * as Nav from 'src/libs/nav'
+import * as Style from 'src/libs/style'
+
+
+const sideMargin = '5rem'
+
+const styles = {
+  top: {
+    container: {
+      height: 490,
+      backgroundImage: `url(${cancerCell})`,
+      backgroundRepeat: 'no-repeat', backgroundSize: '1800px', backgroundPosition: 'right -120px top -95px',
+      backgroundColor: 'black', color: 'white',
+      padding: `7.5rem 0 0 ${sideMargin}`
+    },
+    title: {
+      textTransform: 'uppercase',
+      fontSize: 16, fontWeight: 500
+    },
+    summary: {
+      fontSize: 40, fontWeight: 500, lineHeight: '52px',
+      width: 600, margin: '0.5rem 0 3rem'
+    },
+    details: {
+      fontSize: 16, textTransform: 'uppercase'
+    }
+  },
+  banner: {
+    container: {
+      display: 'flex', alignItems: 'center',
+      backgroundColor: '#170e43', color: 'white',
+      padding: `1.5rem ${sideMargin}`
+    },
+    textContainer: {
+      flex: 1
+    },
+    title: {
+      fontSize: 24, fontWeight: 500
+    },
+    details: {
+      fontSize: 24
+    },
+    buttonContainer: {
+      flex: 'none'
+    }
+  },
+  content: {
+    container: {
+      padding: `3rem ${sideMargin}`
+    },
+    header: {
+      ...Style.elements.sectionHeader, textTransform: 'uppercase'
+    },
+    title: {
+      fontSize: 20, color: Style.colors.secondary
+    }
+  }
+}
+
+
+const FadeBox = ({
```

Comment:
I started to make this as generic as possible, seeing that it's in other designs such as the home page, but then I noticed that they're critically different--there's no border line, so you can do them with one div. So maybe this won't be reused anywhere.